content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -52,34 +52,33 @@ rules:
 
 Alpha metric →  Stable metric →  Deprecated metric →  Hidden metric → Deletion
 
-Alpha metrics have no stability guarantees; as such they can be modified or deleted at any time.
+Alpha metrics have no stability guarantees. These metrics can be modified or deleted at any time.
 
-Stable metrics can be guaranteed to not change; Specifically, stability means:
+Stable metrics are guaranteed to not change. This means:
+* A stable metric will not be deleted or renamed
+* A stable metric's type will not be modified
 
-* the metric itself will not be deleted (or renamed)
-* the type of metric will not be modified
+Deprecated metrics are slated for deletion, but are still available for use.
+These metrics will inlude an annotation about the version in which they became deprecated.
 
-Deprecated metric signal that the metric will eventually be deleted; to find which version, you need to check annotation, which includes from which kubernetes version that metric will be considered deprecated.
-
-Before deprecation:
-
-```
-# HELP some_counter this counts things
-# TYPE some_counter counter
-some_counter 0
-```
-
-After deprecation:
-
-```
-# HELP some_counter (Deprecated since 1.15.0) this counts things
-# TYPE some_counter counter
-some_counter 0
-```
-
-Once a metric is hidden then by default the metrics is not published for scraping. To use a hidden metric, you need to override the configuration for the relevant cluster component.
-
-Once a metric is deleted, the metric is not published. You cannot change this using an override.
+For example:
+* Before deprecation
+  ```
+  # HELP some_counter this counts things
+  # TYPE some_counter counter
+  some_counter 0
+  ```
+
+* After deprecation
+  ```
+  # HELP some_counter (Deprecated since 1.15.0) this counts things
+  # TYPE some_counter counter
+  some_counter 0
+  ```
+
+Hidden metrics are no longer published for scraping, but are still available for use. To use a hidden metric, override the configuration for the relevant cluster component.
+
+Deleted metrics are no longer published and cannot be used. There is no override available to change this.
```

Comment:
Do we need the language about not being able to override this here?

Reply:
Since I'm no longer referring to overrides in the hidden metrics bit before it, I'll remove it here too.